medicines/pars-upload/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "semi": false,
```

Comment:
nice

Reply:
would have been nice as a separate PR, though – lots of changes to comb through in this part-linting, part-functional review

Reply:
Yeah, I should have put this in a separate PR but I just snapped at one point and had to sort the formatting out 😄